lib/api/web-element/commands/getProperty.js

```diff
@@ -37,9 +37,9 @@
  * @param {string} name element property
  * @instance
  * @syntax browser.element(selector).getProperty(name)
- * @see https://www.w3.org/TR/webdriver#get-element-property
+ * @link /#get-element-property
  * @returns {ScopedValue<string>}
  */
-module.exports.command = function(name) {
+module.exports.command = function (name) {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.